File: app/src/test/java/com/mgaetan89/showsrage/fragment/SettingsServerFragmentTest.java
package com.mgaetan89.showsrage.fragment;

import com.mgaetan89.showsrage.R;

import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class SettingsServerFragmentTest {
	@Test
	public void getTitleResourceId() {
		assertThat(new SettingsServerFragment().getTitleResourceId()).isEqualTo(R.string.server);
	}

	@Test
	public void getXmlResourceFile() {
		assertThat(new SettingsServerFragment().getXmlResourceFile()).isEqualTo(R.xml.settings_server);
	}
}

File: app/src/test/kotlin/com/mgaetan89/showsrage/fragment/SettingsServerFragmentTest.kt
package com.mgaetan89.showsrage.fragment

import com.mgaetan89.showsrage.R
import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class SettingsServerFragmentTest {
	@Test
	fun getTitleResourceId() {
		assertThat(SettingsServerFragment.newInstance().getTitleResourceId()).isEqualTo(R.string.server)
	}

	@Test
	fun getXmlResourceFile() {
		assertThat(SettingsServerFragment.newInstance().getXmlResourceFile()).isEqualTo(R.xml.settings_server)
	}
}
